bin/start-coordinator.sh

```diff
@@ -60,15 +60,16 @@ set -u
 
 echo "class path is $CLASSPATH"
 
-JVM_ARGS=" -server \
-          -Xmx${XMX_SIZE:-8g} \
-          -Xms${XMX_SIZE:-8g} \
+UNIFFLE_COORDINATOR_XMX_SIZE=${UNIFFLE_COORDINATOR_XMX_SIZE:-${XMX_SIZE:-"8g"}}
```

Comment:
How to define different XMX_SIZE for server and coordinator dashboard on
same node?

Reply:
Yeah, we deploy `coordinator and server` in different node too in the production env, but I start all services of coordinator in our test env and local laptop in same node, this can make it easy and welcome to new contributor of uniffle.